pkg/cloudprovider/providers/gce/gce_clusters.go

```diff
@@ -37,21 +40,43 @@ func (gce *GCECloud) ListClusters(ctx context.Context) ([]string, error) {
 	return allClusters, nil
 }
 
+func (gce *GCECloud) GetClusters(ctx context.Context) ([]*container.Cluster, error) {
```

Comment:
Can the name be more specific? It's not all clusters. GetManagedClusters?

Reply:
Fixed GetClusters to be GetManagedClusters. Leaving ListClusters as ListClusters as it is part of the Cloud Clusters interface, despite also being a "managed" call.